hack/ci/openstack.sh

```diff
@@ -157,6 +168,9 @@ function cloud_cleanup {
         if ! openstack server delete --wait "$name"; then
             openstack server show "$name" && exit 1
         fi
+        if ! openstack volume delete "$name"; then
+            openstack volume show "$name" && exit 1
```

Comment:
just curious , this is CI nested KVM env, should be deleted (the GCP VM) 

so the reason for explicit delete volume is ? for local test?

Reply:
No this is for setting up the devstack on Openstack (instead of GCP). I'm running this against an Openstack cloud when developing e2e tests. It creates the servers for running devstack that is then used for the e2e tests.

I use it like this:
```bash
# In the normal CI we have RESOURCE_TYPE=gce-project
export RESOURCE_TYPE=openstack
./hack/ci/create_devstack.sh
make test-e2e
./hack/ci/create_devstack.sh cleanup
# After this the volumes for the devstack servers were still there.
```